core/store/migrate/migrations/0137_remove_tx_index.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+DROP INDEX log_broadcasts_unique_idx;
```

Comment:
Maybe add `IF EXISTS` on both drops to be safe :shrug:
```suggestion
DROP INDEX IF EXISTS log_broadcasts_unique_idx;
```